signalfx-tracing/signalfx-dotnet-tracing/cloudfoundry-buildpack/README.md

```diff
@@ -0,0 +1,42 @@
+# CloudFoundry deployment with SignalFx Tracing Library for .NET
+
+This is an example of automatically producing traces using the
+[SignalFx Tracing Library for .NET](https://github.com/signalfx/signalfx-dotnet-tracing/).
+This example is a simple web app that just executes an HTTP call when its only endpoint is called.
+It's auto-instrumented by the SignalFx Tracing Library for .NET Buildpack.
+
+## Building the example app
+
+To run this example locally and send traces to your available Smart Agent or OpenTelemetry Collector,
```

Comment:
It will be good to list the necessary tools. Is it possible to do the same on Windows?

Reply:
> Is it possible to do the same on Windows?

Honestly I have no idea, and I have no way to test it. Probably yes - both `dotnet` and `cfcli` have Windows releases.